formalization_of_commits.py

```diff
@@ -0,0 +1,66 @@
+import requests

```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 REQUESTS-1013 Updated formalization_of_commits.py 
 Mistakes: 
 Wait...What did you do(Updated)? It is not from ['Added', 'Deleted'].